docker/Dockerfile.api

```diff
@@ -7,28 +13,20 @@ COPY /packages /app/packages
 COPY package.json /app
 COPY package-lock.json /app
 
-RUN apk add --no-cache python3 make g++ libc6-compat postgresql-dev
+RUN npm install
```

Comment:
Is there a reason for using `npm install` instead of `npm ci` in the docker files?